linker/shared/src/main/scala/org/scalajs/linker/standard/LinkerBackend.scala

```diff
@@ -33,6 +33,15 @@ abstract class LinkerBackend {
   /** Symbols this backend needs to be present in the linking unit. */
   val symbolRequirements: SymbolRequirement
 
+  /** Additional IR files to inject for linking, mandated by this back-end.
+   *
+   *  Example: the standard emitter back-end injects `RuntimeLong.sjsir` and
+   *  its companion object, unless it uses `BigInt`s to implement `Long`s.
+   *
+   *  The default implementation in `LinkerBackend` returns an empty sequence.
+   */
+  def injectedIRFiles: Seq[IRFile] = Nil
```

Comment:
We should consider making this abstract: It took me quite a while to debug that a forwarder like this was forgotten in a test. I'd expect most real life usages to run into a similar issue.

Reply:
Yes, making it abstract sounds good.